lib/webapi/webapi.go

```diff
@@ -1490,7 +1490,9 @@ func newWebCluster(cluster ops.Site, releases []webRelease, shallow bool) webClu
 	// This significantly reduces amount of traffic b/w frontend and server and
 	// improves the web application performance.
 	if shallow {
-		webCluster.App = ops.Application{}
+		webCluster.App = ops.Application{
+			Package: webCluster.App.Package,
```

Comment:
Hm, not sure what this does? `webCluster.App.Package` is empty cause you're just assigning to `webCluster.App` for the first time here?

Reply:
It's not empty and we are overriding an existing `App` value with the new object. This object has all empty fields except the `Package` which we preserve.

Reply:
Ah, the `App` field comes from the embedded `Site`. Makes sense then, the variable declaration above threw me off.